go.sum

```diff
@@ -272,6 +272,8 @@ github.com/vmware/vra-sdk-go v0.2.0 h1:HbS8C6zv9miMkAUVfnulzKDtJ/Q4naaqt0wpLzDk+
 github.com/vmware/vra-sdk-go v0.2.0/go.mod h1:dqUEePGPUHNYQ/p9DKZM1amqB2WP1uI9JkYvzF+auVo=
 github.com/vmware/vra-sdk-go v0.2.1 h1:nDqlpc/E6O+PUEv7Y9SVckp9xaALwUdr9KkO9EvWRlU=
 github.com/vmware/vra-sdk-go v0.2.1/go.mod h1:dqUEePGPUHNYQ/p9DKZM1amqB2WP1uI9JkYvzF+auVo=
```

Comment:
Likely need to run "go mod tidy" to remove the older sdk lines.

Reply:
Done.